src/NuGet.Core/NuGet.Frameworks/DefaultFrameworkMappings.cs

```diff
@@ -329,12 +329,15 @@ public IEnumerable<OneWayCompatibilityMappingEntry> CompatibilityMappings
                                 new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.WinRT, new Version(4, 5, 0, 0)))),
 
                         // NetCoreApp projects support NetStandard
-                        new OneWayCompatibilityMappingEntry(new FrameworkRange(
-                            new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.NetCoreApp, new Version(1, 0, 0, 0)),
-                            new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.NetCoreApp, FrameworkConstants.MaxVersion)),
-                            new FrameworkRange(
-                                FrameworkConstants.CommonFrameworks.NetStandard10,
-                                FrameworkConstants.CommonFrameworks.NetStandard15))
+                        CreateStandardMapping(
+                            FrameworkConstants.CommonFrameworks.NetCoreApp10,
+                            FrameworkConstants.CommonFrameworks.NetStandard16),
+
+                        // NetCoreApp projects support NetStandard
```

Comment:
Yup! This change already went in so I will address this change separately. 
http://i.imgur.com/sVYTXTe.jpg